easybuild/easyconfigs/p/pestpp/pestpp-5.0.5-foss-2020a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Custom
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'pestpp'
```

Comment:
@robqiao Shouldn't we change this to `PEST++`, which is the official software name?

Reply:
Hmm, this would be a hard call. The community refer it as `pestpp` typically and all functions are labelled as `pestpp-*`. 

Alright, let's adopt the notion used in the documentation as `PEST++` just for consistancy sick.